packages/builders/src/components/selectMenu/SelectMenuOption.ts

```diff
@@ -8,23 +8,61 @@ import {
 } from '../Assertions';
 
 /**
- * Represents a validated option within a select menu component
+ * Represents a option within a select menu component
  */
-export class SelectMenuOptionBuilder extends UnsafeSelectMenuOptionBuilder {
-	public override setDescription(description: string) {
-		return super.setDescription(labelValueValidator.parse(description));
+export class SelectMenuOptionBuilder {
+	public constructor(public data: Partial<APISelectMenuOption> = {}) {}
+
+	/**
+	 * Sets the label of this option
+	 * @param label The label to show on this option
+	 */
+	public setLabel(label: string) {
+		this.data.label = label;
```

Comment:
This should validate the label

Reply:
This is done https://github.com/imranbarbhuiya/discord.js/blob/03ef3873c27eaaa97c1bdba2b0acf35ac0830f08/packages/builders/src/components/selectMenu/SelectMenuOption.ts#L62
here. It was like this before so I kept it as it was.

Reply:
Well its also done there to ensure people actually called the set methods that are required. We always want to validate the input the users give in each set method